### PR TITLE
Bryn-Lewis-Speaker-Profile

### PR DESCRIPTION
**Pull request by KiwiBryn**
Hopefully level of detail is suitable..

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=641) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=641) before we can accept your contribution.<br/><br/>:x: KiwiBryn [sign now](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=641)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-foundation/website?pullRequest=641) it.</sub>

File: input/community/speakers/bryn-lewis.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Location: Christchurch, New Zealand
+Email: bryn.lewis@devmobiloe.co.nz
+Language:
+  - English
+Topics:
+  - Azure
+  - IoT
+Blog: https://blog.devmobile.co.nz/
+Twitter: https://twitter.com/kiwibryn
+LinkedIn:https://linkedin.com/in/brynlewis
```
Comment (SeanKilleen) [inline input/community/speakers/bryn-lewis.md:12]:
```suggestion
LinkedIn: https://linkedin.com/in/brynlewis
```

The build is currently choking on this, I believe.

File: input/community/speakers/bryn-lewis.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+When the device might never get updated and there are definitely no Containers or patch Tuesday.
+
+I get these tiny devices to “play nice” with Azure over wireless technolgies like LoRa, LoRaWAN and other proprietary protocols.
+
+I also work with the little .Net platforms like the nanoFramework which is now a .NET foundation member, the TinyCLR from GHI Electronics and the Meadow from Wilderness labs.
+
+I have a sideline in shifting "legacy to the cloud" mid 90's tech like MSMQ, Classic ASP, COM+, and C++ interop.
+
+
+
```
Comment (SeanKilleen) [inline input/community/speakers/bryn-lewis.md:35]:
```suggestion
```

Suggest removing white space at the end of the file.

**Review by SeanKilleen — COMMENTED** [review]
Minor suggestions to help the build pass

**Pull request merged**